src/Extensions/Abstractions/ServiceContext/OmexServiceContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+
+namespace Microsoft.Omex.Extensions.Abstractions
+{
+	/// <summary>Information about service context</summary>
+	public abstract class OmexServiceContext : IServiceContext
+	{
+		/// <summary>Creates an instance of OmexServiceContext</summary>
+		protected OmexServiceContext(Guid partitionId, long replicaOrInstanceId)
```

Comment:
>Guid partitionId, long replicaOrInstanceId [](start = 31, length = 42)

Missing `<param>` comments

Reply:
Comments visible here https://github.com/microsoft/Omex/blob/feature/create-omex-extensions/src/Extensions/Abstractions/ServiceContext/OmexServiceContext.cs
but I've failed to put any useful information into it